docs/whitepapers/kalmanfilter/kalmanfilter.tex

```diff
@@ -0,0 +1,345 @@
+
+\begin{equation}
+\begin{bmatrix}
+z_{t+1}\\
+v_{z,t+1}
+\end{bmatrix} =
+\begin{bmatrix}
+1 & T_s \\
+0 & 1 
+\end{bmatrix}
+\begin{bmatrix}
+z_t\\
+v_{s,t}
+\end{bmatrix} + 
+\begin{bmatrix}
+{1/2}{T^2}_s & {1/2}{T^2}_s \\
+T_s & T_s \\ 
+\end{bmatrix}
+\begin{bmatrix}
+a_{T,t}^z \\ 
+-g
+\end{bmatrix},
+\end{equation}
+\subsection{state vector definition}
+The state vector \( x_t \) is defined as:
+\begin{equation}
+    x_t = 
+    \begin{bmatrix}
+        z_t &
+        V_{t} & 
+        \alpha_t & 
+        \beta_t &
+        bias_t 
+    \end{bmatrix}
+\end{equation}
+
+\subsection{State Transition Equation}
+\begin{equation}
+    {x}_{t+1} = A_t {x}_t + W_t
+\end{equation}
+
+\subsection{Measurement Model}
+The measurement model is defined as:
+
+\begin{equation}
+y_t = C_t x_t + V_t
+\end{equation}
+\begin{equation}
+    y_t  = 
+    \begin{bmatrix}
+        1 & 0 & 0 & 0 & 0 \\
+        1 & 0 & 0 & 0 & 0 \\
+        1 & 0 & 0 & 0 & 1 \\
+    \end{bmatrix}
+    + x_t + V_t
+\end{equation}
+\\
+
+\noindent
+\section{Translational dynamics}
+
+The movement dynamics of an unmanned aerial vehicle (UAV) in the horizontal
+plane are characterized by its translation in the x and y coordinates. 
+These dynamics share similarities with the UAV's ascent dynamics, 
+excluding the persistent negative acceleration component that's factored 
+in during altitude modeling but not here.
+\\
+
+\noindent
+Similar to its vertical counterpart, the UAV's discrete-time horizontal 
+dynamics are governed by the following equations:
+
+\begin{equation}
+\begin{bmatrix}
+    x_{t+1} \\
+    v_{x,t+1} \\
+    y_{t+1} \\
+    v_{y,t+1} \\
+\end{bmatrix} =
+\begin{bmatrix}
+    1 & T_s & 0 & 0 \\
+    0 & 1 & 0 & 0 \\
+    0 & 0 & 1 & T_s \\
+    0 & 0 & 0 & 1 \\
+\end{bmatrix}
+\begin{bmatrix}
+    x_t \\
+    v_{x,t} \\
+    y_t \\
+    v_{y,t} \\
+\end{bmatrix} +
+\begin{bmatrix}
+    \frac{1}{2}T_s^2 & 0 \\
+    T_s & 0 \\
+    0 & \frac{1}{2}T_s^2 \\
+    0 & T_s \\
+\end{bmatrix}
+\begin{bmatrix}
+    a_{x}^u \\
+    a_{y}^u \\
+\end{bmatrix},
+\end{equation}
+
+\begin{equation}
+    Y^{x,y} = 
+\begin{bmatrix}
+    1 & 0 & 0 & 0 \\
+    0 & 0 & 1 & 0 \\
+\end{bmatrix}
+X_t^{x,y}
+\end{equation}
+where \( x_t \) and \( y_t \) represent the drone's horizontal position,
+ and \( v_{x,t} \) and \( v_{y,t} \) are its horizontal velocity 
+ components along the x and y axes, measured in meters per second. 
+ The terms \( a_{x}^u \) and \( a_{y}^u \) signify the control input 
+ accelerations in the horizontal plane, with units of meters per second 
+ squared. The variable \( T_s \) denotes the interval of the sampling 
+ time in seconds. It is important to note that the term "position" in 
+ this context refers to the center of mass coordinates of the Quadcopter in the 
+ horizontal plane unless otherwise stated.
+\section{Controler design}
```

Comment:
1. Controller
2. Remove this section; this whitepaper is about sensor fusion only